packages/utils/crates/utils-types/src/primitives.rs

```diff
@@ -49,6 +50,14 @@ impl Address {
         self.to_hex()
     }
     // }
+
+    /// Creates a random Ethereum address, mostly used for testing
+    pub fn random() -> Self {
+        let mut addr = [0u8; Self::SIZE];
+        getrandom(&mut addr[..]).unwrap();
+
+        Self { addr }
```

Comment:
```suggestion
        Self { random_bytes() }
```

Please do not use `getrandom` directly, always use one of the functions from the `random` module from `core-crypto`

Reply:
That doesn't work here because adding `core-crypto` as a dependency creates a circular dependency